pyproject.toml

```diff
@@ -54,6 +54,7 @@ fsspec = "^2023.12.2"
 fastparquet = "^2023.8.0"
 graspologic = "^3.3.0"
 plotly = "*"
+simplification = "^0.7.10"
```

Comment:
I can't see this package in conda-forge/bioconda - so I think we should think twice before brining it in.